src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -96,23 +110,37 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime советуют использовать при записи даты в базу данных.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, ZoneOffset.of("+02:00"));
     }
 
     /**
      * Дана строка в виде "yyyymmdd".
      * Необходимо вернуть Optional даты в LocalDate формате
      */
     public Optional<LocalDate> parseDate(String date) {
-        return Optional.empty();
+        Optional<LocalDate> result = Optional.empty();
+        try {
+            result = Optional.ofNullable(LocalDate
+                    .parse(date, DateTimeFormatter.ofPattern("yyyyMMdd")));
```

Comment:
```suggestion
                    .parse(date, DateTimeFormatter.BASIC_ISO_DATE));
```
you can use existing formatter here